src/fundus/scraping/crawler.py

```diff
@@ -179,12 +185,15 @@ def crawl(
             Iterator[Article]: An iterator yielding objects of type Article.
         """
 
-        if max_articles == 0:
+        if max_articles == 0 or max_seconds == 0:
             return
 
         if max_articles is None:
             max_articles = -1
 
+        if max_seconds is None:
+            max_seconds = math.inf
```

Comment:
`math.inf` is of type `float`. To resolve the typing issue I would suggest setting `max_seconds` to -1 here.

Reply:
Thanks, have updated 👍